packages/theme-a11y/theme-a11y.test.js

```diff
@@ -116,15 +119,19 @@ describe('bindInPageLinks()', () => {
       '<a id="link" href="#title"></a>' +
       '<a id="invalidLink1" href="#"></a>' +
       '<a id="invalidLink2" href="/otherlink"></a>' +
-      '<h1 id="title">Title</h1>';
+      '<a id="ignoredLink" class="js-ignore-link" href="#title2"></a>' +
+      '<h1 id="title">Title</h1>' +
+      '<h2 id="title2">Title 2</h2>';
   });
 
   test('is a function exported by theme-a11y.js', () => {
     expect(typeof bindInPageLinks).toBe('function');
   });
 
-  test('returns array of link elements that were binded', () => {
-    const links = bindInPageLinks();
+  test('returns array of link elements that were binded and not ignored', () => {
+    const links = bindInPageLinks({
```

Comment:
Not too familiar with tests, not sure it's best to test 2 things (if `bindInPageLinks` works and if ignore param works) ? Should it be separated? What do you think @t-kelly

Reply:
Fine to test two things, but your `expects` below don't reflect that. Maybe include something like:
```
expect(links).not.toContain(document.queryElement('.js-ignore-link');
```

Reply:
Ended up going with [expect.not.arrayContaining(array)](https://jestjs.io/docs/en/expect#expectnotarraycontainingarray).

If we wanted to use `not.toContain`, the query would have been on an `id`. Resulting in something like : 

```HTML
<a id="ignoredLink" class="js-ignore-link" href="#title2"></a>
```
```JS
const links = bindInPageLinks({
  ignore: '.js-ignore-link'
});

expect(links).not.toContain(document.queryElement('#ignoreLink');
```

It bothered me that the ignore selector wasn't the same one being tested.